README.md

```diff
@@ -14,9 +14,9 @@ Browser automation is the act of executing actions automatically in a web browse
 
 ## Tools
 
-:lock: - Private  
-:moneybag: - Paid  
-:no_entry_sign: - Not Maintained  
+:lock: - Private
```

Comment:
I'm not sure of the reasoning behind this change,but the list is a bit harder to read this way, please add the trailing spaces back

Reply:
Ah, that's my editor - it's removing trailing spaces automatically... One second

Reply:
Done @angrykoala